build/localnet/cluster-manager/main.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/pokt-network/pocket/runtime/defaults"
+	"github.com/pokt-network/pocket/shared/crypto"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+var (
+	validatorKeysMap = make(map[string]crypto.Ed25519PrivateKey)
```

Comment:
1.  🫡
2. I wouldn't rename now since this is very much code related to auto scaling validators. When we'll handle other actors I'd refactor accordingly